Source/ACE/Network/Enum/Useability.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace ACE.Network.Enum 
+{
+    public enum Useability : uint
+    {
+        USEABLE_UNDEF                                           = 0,
```

Comment:
Convert to hex and sort by number please.  Also switch to ProperCase for element names.